src/software/full_system_main.cpp

```diff
@@ -89,10 +90,11 @@ int main(int argc, char **argv)
             ai->Subject<AIDrawFunction>::registerObserver(visualizer);
             ai->Subject<PlayInfo>::registerObserver(visualizer);
             backend->Subject<SensorProto>::registerObserver(visualizer);
+            backend->Subject<TbotsProto::PrimitiveSet>::registerObserver(visualizer);
```

Comment:
Should simplify things:
```suggestion
            ai->Subject<TbotsProto::PrimitiveSet>::registerObserver(visualizer);
```